docs/Features-for-Go-in-VS-Code.md

```diff
@@ -4,20 +4,23 @@
 - Signature Help for functions as you type (using `gogetdoc` or `godef`+`godoc`)
 - Quick Info on the symbol as you hover over it (using `gogetdoc` or `godef`+`godoc`)
 
+If `go.useLanguageServer` is enabled, most of the above features will be
+offered through the Go language server `gopls`.
```

Comment:
Why just above? Wouldnt some of the below features be offered via gopls as well?